src/System.Private.Xml/tests/Xslt/XslTransformApi/CXslTransform.cs

```diff
@@ -498,17 +488,10 @@ public void XmlResolver3(InputType inputType, ReaderType readerType, TransformTy
         [InlineData(InputType.Navigator, ReaderType.XmlValidatingReader, TransformType.Writer, DocType.XPathDocument)]
         [InlineData(InputType.Navigator, ReaderType.XmlValidatingReader, TransformType.Stream, DocType.XPathDocument)]
         [InlineData(InputType.Navigator, ReaderType.XmlValidatingReader, TransformType.TextWriter, DocType.XPathDocument)]
-        [Theory(Skip = "When style sheet URI = Intranet zone, XmlSecureResolver does not resolve document function")]
-        public void XmlResolver7(InputType inputType, ReaderType readerType, TransformType transformType, DocType docType)
+        [Theory(Skip = "Absolute path is hard coded, it needs to be set dynamically")]
```

Comment:
What's the plan for re-enabling the test?  Is there an associated issue # that should be included in the message?  Also, we've generally not been using Skip to cause tests to be skipped, instead using [ActiveIssue(12345)].